### ISSUE TITLE
Hard to tell which account I am logged in to

### ISSUE THREAD

**Issue by jidanni** [kartaview/openstreetcam.org#333]
One cannot tell if one is logged in to one's ralph account, or one's ralph2 account.
![20210619T032918](https://user-images.githubusercontent.com/1230959/122608361-a8371080-d0ae-11eb-86ed-22519c413b90.jpg)
It would be best to have the user's name appear here.

**Comment by miruna-craciun** [comment]
@jidanni - if you hover over the user initial icon, you will see the username in the lower-left corner. 
You can also tell which account you're using by the URL. 

<img width="1440" alt="Screenshot 2021-06-28 at 14 06 54" src="https://user-images.githubusercontent.com/77106534/123627044-2ebfc080-d81a-11eb-89dc-47ca651fefe7.png">

**Comment by jidanni** [comment]
Yes, it is true it is possible to detect who I am. I'm just saying that I can't think of one other site that doesn't at least have a "Hello Nibblesworth" in the corner.
